index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/font-awesome/css/font-awesome.min.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <nav class="flex-nav">
+        <h1 class="logo">My Logo</h1>
+        <div class="toggleNav"><i class="fa fa-bars"></i></div>
```

Comment:
Kindly use a link (`a` tag). for the Logo
![image](https://github.com/lukademinnosa/module1_project1/assets/78251927/a057964e-7e4b-4fc0-a57d-684532c56fc0)

* This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

Your design
![image](https://github.com/lukademinnosa/module1_project1/assets/78251927/dd50d5fa-f6a3-44a9-a8e4-2f33c6ee3556)

Figma
![image](https://github.com/lukademinnosa/module1_project1/assets/78251927/668a21a0-d935-4768-82bf-a17bc967659f)

Please kindly adjust your design to ensure that our website looks exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

- The missing hamburger menu
- The color of the link `My Logo`